landlab/components/groundwater/dupuit_percolator.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+"""
+GroundwaterDupuitPercolator Component
+
+@author: G Tucker
+"""
+
+import numpy as np
+from landlab import Component
+from landlab.utils import return_array_at_node, return_array_at_link
+from landlab.grid.mappers import map_mean_of_link_nodes_to_link
+
+
+ACTIVE_LINK = 0
+
+
+class GroundwaterDupuitPercolator(Component):
+    """
+    Simulate groundwater flow in a shallow unconfined aquifer.
+
+    The GroundwaterDupuitPercolator uses the Dupuit approximation that the
+    hydraulic gradient is equal to the slope of the water table.
+
+    Parameters
+    ----------
+    grid: ModelGrid
+            Landlab ModelGrid object
```

Comment:
I'd put the parameters only in one place. Just after the init would be my preference (e.g., not here).

Reply:
We need a standard... after __init__ is fine with me.